haproxy/service.sls

```diff
@@ -1,7 +1,9 @@
+{% from "haproxy/map.jinja" import haproxy with context %}
+
 haproxy.service:
 {% if salt['pillar.get']('haproxy:enable', True) %}
   service.running:
-    - name: haproxy
+    - name: {{ haproxy.service }}
```

Comment:
Hey @daks,  This all looks good,  Just one question ...  If os_family is FreeBSD will **`haproxy.service`** be None?   There is unfortunately no defaults.yaml so formula requires mandatory pillars on FreeBSD?  Same question regarding package, etc

Reply:
In map.jinja only parameter defined is `group` ..

Reply:
In map.jinja, the merge uses `base='default'` to merge with default dict which itself contains `service: haproxy` so for FreeBSD (or others) `service` will be set to `haproxy`